index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet">
+    <!-- <script src="./src/index.js"></script> -->
+    <script src="./node_modules/axios/dist/axios.min.js"></script>
     <title>Weather Report</title>
 </head>
-<body>
-    
+<body id="background">
+    <header>
+        <h1> Sana's Swiftie Weather Report </h1>
+    </header>
+
+    <section id="location">
+        <button id="reset" class="clicky">
+            Reset
+        </button>
+
+        <form id="search">
+            <input type="text" id="city" class="box" value="London">
+            <button id="go" class="clicky" type="submit">
+                Go
+            </button>
+        </form>
+        
+
+        <select id="skySelect" class="box">
+            <option>Lover</option> 
+            <option>rep</option>
+            <option>1989</option> 
+            <option>Red</option>
+            <option>Speak Now</option>
+            <option>fearless</option> 
+
+        </select>
+    </section>
```

Comment:
I think that these elements could be nested with the `main` element since this is content is a main feature of your weather report page.